src/test/java/utils/DriverSetUp.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.io.*;
+import java.util.Properties;
+
+public class DriverSetUp {
+    protected static WebDriver driver;
+    private String filePath = "/Users/avoloshchuk/IdeaProjects/com.sharelane/src/test/resources/app.properties";
+    private Properties appProps = new Properties();
+
+    public void loadPropertiesFromFile() {
```

Comment:
this method could be as a private one

Reply:
Access modifier of loadPropertiesFromFile() method was changed to private.